packages/matchbox/src/components/Tooltip/Tooltip.js

```diff
@@ -1,124 +1,166 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
 import TooltipOverlay from './TooltipOverlay';
+import { Box } from '../Box';
 
-import styles from './Tooltip.module.scss';
+import styled from 'styled-components';
+import { visibility } from './styles';
+import { deprecate } from '../../helpers/propTypes';
+import { border, shadow, background, padding, typography, compose } from 'styled-system';
+import { pick } from '@styled-system/props';
+import { createPropTypes } from '@styled-system/prop-types';
 
-class Tooltip extends Component {
-  static displayName = 'Tooltip';
+const system = compose(border, shadow, background, padding, typography);
 
-  static propTypes = {
-    content: PropTypes.node,
-    /**
-     * Disables hover events
-     */
-    disabled: PropTypes.bool,
-    dark: PropTypes.bool,
-    active: PropTypes.bool,
-    left: PropTypes.bool,
-    right: PropTypes.bool,
-    top: PropTypes.bool,
-    bottom: PropTypes.bool,
-    horizontalOffset: PropTypes.string,
-    /**
-     * Disables automatic positioning
-     */
-    forcePosition: PropTypes.bool,
-    children: PropTypes.node,
-    /**
-     * These props are provided automatically through the Overlay component
-     */
-    preferredDirection: PropTypes.shape({
-      top: PropTypes.bool,
-      bottom: PropTypes.bool,
-      left: PropTypes.bool,
-      right: PropTypes.bool
-    }),
-    /**
-     * Element ID for the portal that will house tooltips. Appends to body if not provided.
-     */
-    portalId: PropTypes.string
-  };
+const StyledTooltipContainer = styled(Box)`
+  ${visibility}
+`;
 
-  static defaultProps = {
-    right: true,
-    bottom: true,
-    horizontalOffset: '0px',
-    forcePosition: false,
-    preferredDirection: {
-      bottom: true,
-      left: false,
-      right: true,
-      top: false
-    }
-  }
+const StyledContent = styled(Box)`
```

Comment:
Need to change this – using `Box` allows *all* system props, should use a `'div'` instead